docs/source/running-jobs/environment-variables.rst

```diff
@@ -121,30 +121,31 @@ Environment variables for ``run``
    :widths: 20 70 10
    :header-rows: 1
```

Comment:
For consistency, use 2 spaces like we have for the table above?

```suggestion
  :widths: 20 70 10
  :header-rows: 1
```

Reply:
Consistent now.

Reply:
Rest of our docs use two or four spaces for indentation under all sphinx directives (`.. <directive>`). Since there doesn't seem to be a [definitive style guide](https://stackoverflow.com/a/48313531) for Sphinx and multiples of two or four are easier to indent for, let's stick to our current convention instead of changing it in this doc?

Reply:
Should've typed a longer response. I checked all `.. list-table::` in `*rst`: 2 out of 3 indented with the first letter after `.. [l]ist-table::`, so made all 3 align like that. It looks better to me, rather than counting the spaces. 

There are a lot more other directives like `.. code-block:: console` or `.. tip::` which use various indents; we can align those on-demand.

Reply:
Agreed visually 3 spaces is appealing for sphinx, but not sure if visual appeal matters here since this is not user facing. Functionally, it's more error prone since you can't simply press tab to get the desired number of spaces. Also adds mental overhead when switching from writing python code/yaml to writing docs. Either way, super minor, approving now.